TypeScriptReact.YAML-tmLanguage

```diff
@@ -0,0 +1,720 @@
+# Similar to TypeScript.YAML-tmLanguage except:
+#   - Remove type assertion (cast.expr.ts)
```

Comment:
Haven't found a good way to do this. I can move independent scopes like #string, #keyword, #regex, #comment, #operator to a shared common syntax file. This can save ~100 lines each file. I cannot do the same thing with scopes that involves #expression because of its recursive structure.